test/FixedGovLst.t.sol

```diff
@@ -177,6 +177,33 @@ contract Constructor is FixedGovLstTest {
     assertEq(fixedLst.SHARE_SCALE_FACTOR(), lst.SHARE_SCALE_FACTOR());
     assertEq(fixedLst.name(), tokenName);
     assertEq(fixedLst.symbol(), tokenSymbol);
+    assertEq(fixedLst.decimals(), 18);
+    assertEq(fixedLst.version(), "2");
+  }
+}
+
+contract DefaultDelegatee is FixedGovLstTest {
+  function test_DefaultDelegatee() public view {
+    assertEq(fixedLst.defaultDelegatee(), lst.defaultDelegatee());
+  }
+}
+
+contract DepositForDelegatee is FixedGovLstTest {
+  function testFuzz_DepositForDelegatee(address _delegatee) public view {
```

Comment:
Two thoughts on this test:

* We need a more expressive name for `scopelint`
* The deposit for a given delegatee is not set until it is initialized. So I think what wee need to do is first `fetchOrInitializeDepositForDelegatee` then call `depositForDelegatee` on the fixedLst and make sure it is both nonzero and equal to the rebasing lst

Reply:
Done: 07b5ee8693b866cebd086d8c01a7ca831257fc15